### DISCUSSION TITLE
我的nian所有记本创建时封面和背景都是空白的，不上传图片就没有图片显示，这是正常的吗

### DISCUSSION THREAD

**Discussion by YLWillowDev** [WatsonYao/nian_app#69] [Q&A] [answered]
<img width="548" alt="image" src="https://github.com/WatsonYao/nian_app/assets/161687086/830e3b03-9531-46ee-bdab-d2ee20db1248">
上面是看别人的示例，我的是全空白的

**Comment by WatsonYao** [comment, marked as answer]
参考新手指南即可
